include/openssl/x509v3.h

```diff
@@ -79,23 +79,24 @@ struct v3_ext_ctx;
 
 // Useful typedefs
 
+typedef struct v3_ext_method X509V3_EXT_METHOD;
+
 typedef void *(*X509V3_EXT_NEW)(void);
 typedef void (*X509V3_EXT_FREE)(void *);
 typedef void *(*X509V3_EXT_D2I)(void *, const unsigned char **, long);
 typedef int (*X509V3_EXT_I2D)(void *, unsigned char **);
-typedef STACK_OF(CONF_VALUE) *(*X509V3_EXT_I2V)(
-    const struct v3_ext_method *method, void *ext,
-    STACK_OF(CONF_VALUE) *extlist);
-typedef void *(*X509V3_EXT_V2I)(const struct v3_ext_method *method,
-                                struct v3_ext_ctx *ctx,
-                                STACK_OF(CONF_VALUE) *values);
-typedef char *(*X509V3_EXT_I2S)(const struct v3_ext_method *method, void *ext);
-typedef void *(*X509V3_EXT_S2I)(const struct v3_ext_method *method,
-                                struct v3_ext_ctx *ctx, const char *str);
-typedef int (*X509V3_EXT_I2R)(const struct v3_ext_method *method, void *ext,
+typedef STACK_OF(CONF_VALUE) *(*X509V3_EXT_I2V)(const X509V3_EXT_METHOD *method,
```

Comment:
Some functions/definitions seem to define parameters as:
```
foo (type a,
        int b,
        thing c)
```

While others as:
```foo(type a, int b, thing c)```

Is there a style guide that indicates which is correct, and when, for me to look at?

Reply:
There's a mention parameters style [here](https://github.com/awslabs/aws-lc/blob/main/STYLE.md#parameters), but I don't see specific guidance for line returns between them.

I suspect the convention is to use a single line for the function declaration unless that line gets too long, otherwise place parameters (after the first) each on a separate line.

Reply:
Whatever the style conventions may be, they're not consistently followed nor enforced.